package.json

```
@@ -11,17 +11,20 @@
   },
   "scripts": {
     "lerna": "lerna",
-    "build": "lerna run build",
-    "build:docs": "lerna run --scope @refinitiv-ui/docs build --stream",
-    "build:prod": "lerna run build:prod",
+    "build": "nx run-many --target build --parallel --all",
+    "build:affected": "nx affected:build --parallel",
+    "build:docs": "nx run @refinitiv-ui/docs:build",
+    "build:prod": "nx run-many --target build:prod --all",
```

```suggestion
    "build:prod": "nx run-many --target build:prod --parallel --all",
```

Is it possible to also have `parallel` flag here ?

---

Or maybe we could have a single `parallel` setting in `nx.json` as I see that nx parallel still respects the package dependency graph.

---

This was removed to have a full log for production builds.